packages/admin-ui/src/pages/system/components/Notifications/index.tsx

```diff
@@ -8,33 +8,47 @@ import { InputForm } from "components/InputForm";
 import { ReqStatus } from "types";
 import ErrorView from "components/ErrorView";
 import Ok from "components/Ok";
+import { withToastNoThrow } from "components/toast/Toast";
 
 export default function Notifications() {
   const telegramStatus = useApi.getTelegramStatus();
 
-  const [reqStatus, setReqStatus] = useState<ReqStatus>({});
+  const [reqStatusToken, setReqStatusToken] = useState<ReqStatus>({});
+  const [reqStatusStatus, setReqStatusStatus] = useState<ReqStatus>({});
   const [token, setToken] = useState("");
 
   async function updateTelegramToken() {
     try {
-      setReqStatus({ loading: true });
-      await api.setTelegramToken({ telegramToken: token });
-      setReqStatus({ result: true });
+      setReqStatusToken({ loading: true });
+      await withToastNoThrow(
```

Comment:
You should use the withToast() variant otherwise the error won't be captured by `setReqStatusToken({ error: e });`